alerts/osd.libsonnet

```diff
@@ -0,0 +1,155 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'osd-alert.rules',
+        rules: [
+          {
+            alert: 'CephOSDsDown',
+            expr: |||
+              (count(ceph_osd_up * on(ceph_daemon) ceph_osd_in == 0) > 0) and ((ceph_pg_total - on(job) ceph_pg_clean) > 0)
```

Comment:
@pcuzner @b-ranto I feel if any of these conditions true, OSDs are down. Is this `and` correct here?